$NetBSD: patch-aa,v 1.1 2007/11/07 17:12:30 rillig Exp $

DragonFly says:

src/ipc.cpp: In function `Gobby::Unix::Address <unnamed>::find_gobby_addr()':
src/ipc.cpp:181: error: `getuid' was not declared in this scope
src/ipc.cpp:202: error: `unlink' was not declared in this scope
...

--- inc/ipc.hpp.orig	2007-08-18 18:02:13.000000000 +0200
+++ inc/ipc.hpp	2007-11-07 18:10:00.000000000 +0100
@@ -25,6 +25,7 @@
 # include <set>
 # include <net6/packet.hpp>
 # include <net6/connection.hpp>
+# include <unistd.h>
 # include "unix.hpp"
 # include "gselector.hpp"
 #endif
